physics/GFS_surface_generic_pre.F90

```diff
@@ -86,8 +86,8 @@ subroutine GFS_surface_generic_pre_run (nthreads, im, levs, vfrac, islmsk, isot,
         real(kind=kind_phys), dimension(:), intent(inout) :: sigmaf, work3, zlvl
 
         ! Stochastic physics / surface perturbations
-        real(kind=kind_phys), dimension(:),   intent(out) :: drain_cpl
-        real(kind=kind_phys), dimension(:),   intent(out) :: dsnow_cpl
+        real(kind=kind_phys), dimension(:),   intent(inout) :: drain_cpl
```

Comment:
These two are only defined in this subroutine, but never explicitly set, so 'INOUT' seemed most appropriate.

Reply:
Yes indeed!